pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -61,13 +63,17 @@ func (e *Executor) Execute(sig executor.StopSignal) model.StageStatus {
 	timer := time.NewTimer(timeout)
 
 	e.reportRequiringApproval()
-	e.LogPersister.Info("Waiting for an approval...")
+
+	num := e.StageConfig.WaitApprovalStageOptions.MinApproverNum
+	if num > 1 {
+		e.LogPersister.Infof("Waiting for approval from at least %d users...", num)
+	} else {
+		e.LogPersister.Infof("Waiting for approval from at least %d user...", num)
```

Comment:
same as the comment below, if the `e.StageConfig.WaitApprovalStageOptions.MinApproverNum` is not set, this will print something like `Waiting for approval from at least 0 user...` 👀